src/main/java/com/acoustic/SpringPolandSalaryCalculator/calculatorservice/TaxService.java

```diff
@@ -0,0 +1,48 @@
+package com.acoustic.SpringPolandSalaryCalculator.calculatorservice;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import org.springframework.stereotype.Component;
+
+import com.acoustic.SpringPolandSalaryCalculator.rates.RatesConfigurationProperties;
+import lombok.RequiredArgsConstructor;
+
+
+@Component
+@RequiredArgsConstructor
+public class TaxService implements SalaryCalculatorService {
+
+
+    public static final int MONTHS_NUMBER = 12;
+    private final RatesConfigurationProperties rate;
+
+
+    @Override
+    public BigDecimal apply(BigDecimal grossMonthlySalary) {
+        return (grossMonthlySalary.multiply(BigDecimal.valueOf(MONTHS_NUMBER))
```

Comment:
What do you think about using the ternary operator only for one-lines code? It is hard to read what happens here.

Reply:
done